kyo-bench/src/test/scala/kyo/benchTest/CatsBenchTest.scala

```diff
@@ -1,4 +1,6 @@
-package kyo.bench
+package kyo.benchTest
```

Comment:
Had to move to a separate package otherwise `Registry` for some reason only returns test classes.